src/firmware/app/primitives/BUILD

```diff
@@ -102,14 +102,22 @@ cc_library(
     srcs = ["primitive.c"],
     hdrs = ["primitive.h"],
     deps = [
-        "//firmware/main/app/world:firmware_world",
+        "//firmware/app/world:firmware_world",
     ],
 )
 
 cc_library(
     name = "primitive_manager",
     srcs = ["primitive_manager.c"],
     hdrs = ["primitive_manager.h"],
+    copts = select({
+        # TODO: this should not be dependent on freertos at all
```

Comment:
TODO

Reply:
Can we leave this for now? lots more immediate refactoring going to happen that will remove this

Reply:
Fine with me, just didn't know if it was supposed to happen in this PR or not.